torch/onnx/_internal/fx/passes/decomp.py

```diff
@@ -74,7 +74,7 @@ def _run(self, *args, **kwargs) -> torch.fx.GraphModule:
                 decomposition_table=self.decomposition_table,
                 tracing_mode=tracing_mode,
                 _allow_non_fake_inputs=True,
-                _allow_fake_constant=self.allow_fake_constant,
+                _allow_fake_constant=bool(self.allow_fake_constant),
```

Comment:
Curious why this is needed? Thanks!

Reply:
mypy was complaining because we were assigning a `Optional[bool]` to a `bool`.